dozer-sql/src/pipeline/errors.rs

```diff
@@ -174,6 +181,41 @@ impl From<dozer_types::pyo3::PyErr> for PipelineError {
     }
 }
 
+#[cfg(feature = "onnx")]
```

Comment:
Scope these items in a mod and write one `#[cfg(feature = "onnx")]`

Reply:
fixed!